### ISSUE TITLE
Rotation sometimes fails to report success

### ISSUE THREAD

**Issue by ethanmdavidson** [google/exposure-notifications-verification-server#2021]
### TL;DR
The rotation service sometimes fails to publish the success metric, causing the forward progress alert to fire a false alarm.

Here is a screenshot of the incident metrics, with data missing from 2:30 - 2:45
![missing data causes alert to fire](https://i.imgur.com/L0vX00l.png)

Here are the logs that appear to show successful runs at 2:30 and 2:32. It appears that cloudrun rotated the instance between 2:30 and 2:32, because the labels.instanceId field changes (value in green).
![logs show successful run](https://i.imgur.com/AWzIaHo.png)

**Expected behavior**
A successful run within the last 35 minutes should prevent alert from firing.

**Observed behavior**
The successful runs at 2:30 and 2:32 didn't show up in the metrics.

**Environment**
- MVS

**Additional Information**
It seems that metrics are tied to something instance specific, as there was another incident on april 9 that was very similar to this one. Let me know if you want screenshots of that one too.

I haven't verified that the rotation at 2:30-2:32 was actually successful, I'm just assuming that a 200 response and no errors means everything is good. If there's a more robust way of checking that it was actually successful please let me know.

I think I've also seen the same thing happen with e2e-runner.

**Comment by sethvargo** [comment]
Hey @ethanmdavidson 

The metric will only be incremented if the rotation is actually successful. If you enable DEBUG mode, the logs will show whether rotation actually occurred (there's a data-layer lock). v0.26.0 lowered those locks (https://github.com/google/exposure-notifications-verification-server/pull/1962), so I think this is fixed in the most recent release.

**Comment by ethanmdavidson** [comment]
okay, thanks. I'll close this for now and look into it more after v0.26.0
